CIP-????/README.md

```diff
@@ -0,0 +1,394 @@
+
+## Simple metadata updates
+
+A simple metadata update transaction must only send ADA from the oracle update address to itself and must not mint any tokens, but it may contain any number of inputs and outputs. Otherwise, the transaction is ignored for the purposes of token metadata.
+
+The schema for simple metadata updates in CIP-???? is similar to the CIP-25 schema, but it is nested under `"????".simple_metadata_update` in the transaction metadata object.
+
+```json
+{
+    "????": {
+        "simple_metadata_update": {
+            "<policyId>": {
+                "<tokenName>": {
+                    "<metadataField>": "<metadataValue>"
+                }
+            }
+        }
+    }
+}
+```
+
+To remove a metadata field, set its value explicitly to `null` in the metadata update.
+
+## Regex metadata updates
+
+The schema for regex metadata updates is as follows:
+
+```json
+{
+    "????": {
+        "regex_metadata_update": {
+            "<policyId>": {
+                "<tokenNameRegex>": {
+                    "<metadataField>": "<metadataValue>"
+                }
+            }
+        }
+    }
+}
+```
+
+A regex metadata update transaction must only send ADA from the oracle update address to itself and must not mint any tokens, but it may contain any number of inputs and outputs. Otherwise, it is ignored for the purposes of token metadata.
+
+The only difference from the simple metadata update is that here the token names are defined in terms of PCRE regular expressions (regex). The regex metadata update applies to any previously minted token whose policy ID matches `<policyID>` and whose token name matches the `<tokenNameRegex>` regular expression.
+
+For example, the following metadata update would apply to every Equine pioneer horse between `EquinePioneerHorse05000` and `EquinePioneerHorse05999`:
+
+```json
+{
+    "????": {
+        "regex_metadata_update": {
+            "30ed3d95db1d6bb2c12fc5228a2986eab4553f192a12a4607780e15b": {
+                "^EquinePioneerHorse05\\d{3}$": {
+                    "age": 2
+                }
+            }
+        }
+    }
+}
+```
+
+The regular expression pattern in `<tokenNameRegex>` is defined according to the grammar in:
+
+-   European Computer Manufacturers Association, "ECMAScript Language Specification 5.1 Edition", ECMA Standard ECMA-262, June 2011. Section 15.10. [https://www.ecma-international.org/wp-content/uploads/ECMA-262_5.1_edition_june_2011.pdf](https://www.ecma-international.org/wp-content/uploads/ECMA-262_5.1_edition_june_2011.pdf)
+
+## Tabular metadata updates
+
+Tabular metadata updates use a condensed rectangular format to specify new values for a fixed set of fields for a large number of assets. Specifically, we use the comma-separated values (CSV) format:
+
+```json
+{
+  "????": {
+	  "tabular_metadata_update": {
+	    "<policyId>": "<CsvValue>"
```

Comment:
```suggestion
	  "tabular_metadata_update": {
	    "<policyId>": "<CsvValue>"
	    }
```
The missing closure here makes it think that the text after the JSON block is still JSON and is also highlighted as errors.  It renders properly on GitHub but might not in other environments.

Reply:
Fixed